airflow/models/taskinstance.py

```diff
@@ -617,6 +619,11 @@ def prev_attempted_tries(self) -> int:
     def next_try_number(self) -> int:
         return self._try_number + 1
 
+    @property
+    def operator_name(self) -> str:
+        """@property: use a more friendly display name for the operator, if set."""
+        return self.custom_operator_name or self.operator  # type: ignore
```

Comment:
Why not set this to `-> str | None` and get rid of the ignore?

Reply:
@uranusjr Done :)